builtin/audit/file/backend.go

```diff
@@ -376,3 +413,19 @@ func (b *Backend) Invalidate(_ context.Context) {
 	defer b.saltMutex.Unlock()
 	b.salt.Store((*salt.Salt)(nil))
 }
+
+func (b *Backend) RegisterNodesAndPipeline(broker *eventlogger.Broker, name string) error {
```

Comment:
Mind adding a godoc to this to explain what it's doing? Might as well add to the other implementation as well.

Reply:
Oh it looks like this is explained in the interface. Feel free to disregard, but might as well mention that we're implementing the Backend interface.